boussole/finder.py

```diff
@@ -180,13 +180,27 @@ def match_conditions(self, filepath, sourcedir=None, nopartial=True,
 
         return True
 
+    def append_suffix(self, filepath, suffix):
+        """
+        Change final filename by appending a suffix.
+
+        Args:
+            filepath (str): A file path (relative or absolute).
+            suffix (str): Suffix to append
+
+        Returns:
+            str: Filepath with the new filename.
+        """
+        filename, ext = os.path.splitext(filepath)
+        return '.'.join([filename, suffix, ext[1:]])
+
     def change_extension(self, filepath, new_extension):
         """
         Change final filename extension.
 
         Args:
             filepath (str): A file path (relative or absolute).
-                new_extension (str): New extension name (without leading dot) to
+            new_extension (str): New extension name (without leading dot) to
```

Comment:
I think it is a typo mistake since this is a wrong indent for Sphinx napoleon docstring section, revert this line change.